src/modules/auto-state-on-scroll-bounds.js

```diff
@@ -0,0 +1,187 @@
+/**
+ *  @author Deux Huit Huit
+ *
+ *  Auto state on scroll bounds
+ 
+ 	Container: Element that scrolls and on which the state will be changed.
+ 				It has 4 states, 2 per axis.
+ 		<add class="js-scroll-bounds-ctn" />
+ 		<add data-scroll-bounds-threshold="0.1" />
+ 		
+ 		<add data-x-start-visible-state-add-class="" />
```

Comment:
The state used in change state is not including the visible terme. I Guest it will not work. It was tested with this attribute name?

Reply:
Nope, these comments are lying to you and they should be fixed now! ;)